packages/features/bookings/lib/handleNewBooking/scheduleNoShowTriggers.ts

```diff
@@ -7,28 +10,57 @@ type ScheduleNoShowTriggersArgs = {
   booking: {
     startTime: Date;
     id: number;
+    location: string | null;
   };
   triggerForUser?: number | true | null;
   organizerUser: { id: number | null };
   eventTypeId: number | null;
   teamId?: number | null;
   orgId?: number | null;
+  destinationCalendars?: DestinationCalendar[] | null;
 };
 
 export const scheduleNoShowTriggers = async (args: ScheduleNoShowTriggersArgs) => {
-  const { booking, triggerForUser, organizerUser, eventTypeId, teamId, orgId } = args;
+  const { booking, triggerForUser, organizerUser, eventTypeId, teamId, orgId, destinationCalendars } = args;
+
+  const isDailyVideoLocation = booking.location === DailyLocationType || booking.location?.trim() === "";
+  const isGoogleMeetLocation = booking.location === "integrations:google:meet";
+
+  if (!isGoogleMeetLocation && !isDailyVideoLocation) return;
 
-  // Add task for automatic no show in cal video
   const noShowPromises: Promise<any>[] = [];
 
-  const subscribersHostsNoShowStarted = await getWebhooks({
+  const hostNoShowTriggerEvent = isDailyVideoLocation
+    ? WebhookTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW
+    : WebhookTriggerEvents.AFTER_HOSTS_GOOGLE_MEET_NO_SHOW;
+
+  const guestNoShowTriggerEvent = isDailyVideoLocation
+    ? WebhookTriggerEvents.AFTER_GUESTS_CAL_VIDEO_NO_SHOW
+    : WebhookTriggerEvents.AFTER_GUESTS_GOOGLE_MEET_NO_SHOW;
+
+  const subscribersHostsNoShowStartedPromises = getWebhooks({
+    userId: triggerForUser ? organizerUser.id : null,
+    eventTypeId,
+    triggerEvent: hostNoShowTriggerEvent,
+    teamId,
+    orgId,
+  });
+  const subscribersGuestsNoShowStartedPromises = getWebhooks({
     userId: triggerForUser ? organizerUser.id : null,
     eventTypeId,
-    triggerEvent: WebhookTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW,
+    triggerEvent: guestNoShowTriggerEvent,
     teamId,
     orgId,
   });
 
+  const [subscribersHostsNoShowStarted, subscribersGuestsNoShowStarted] = await Promise.all([
+    subscribersHostsNoShowStartedPromises,
+    subscribersGuestsNoShowStartedPromises,
+  ]);
+
+  // TODO: is this correct?
+  const destinationCalendar = destinationCalendars?.find((cal) => cal.userId === organizerUser.id);
+
```

Comment:
we need the destination calendar of the organizer (that created the google meet url).